authui/src/form.ts

```diff
@@ -7,37 +7,30 @@ import {
   progressEventHandler,
 } from "./loading";
 import { handleAxiosError } from "./error";
+import { Controller } from "@hotwired/stimulus";
 
 // Handle click link to submit form
 // When clicking element with `data-submit-link`, it will perform click on
 // element with `data-submit-form` that contains the same value
 // e.g. data-submit-link="verify-identity-resend" and
 //      data-submit-form="verify-identity-resend"
-export function clickLinkSubmitForm(): () => void {
-  const links = document.querySelectorAll("[data-submit-link]");
-  const disposers: Array<() => void> = [];
-  for (let i = 0; i < links.length; i++) {
-    const link = links[i];
+export class ClickLinkSubmitFormController extends Controller {
```

Comment:
Redesign this controller to be TransferClickController. This controller can reside in `src/click.ts`

```html
<a data-controller="transfer-click" data-action="transfer-click#click" data-transfer-click-click="selector" />
```

The actual selector should be `#verify-identity-resend-button`. Set this ID on that button and remove `data-submit-form="verify-identity-resend"`